site/docs/pages/checkout/checkout.mdx

```diff
@@ -203,9 +206,13 @@ import { OnchainKitProvider } from '@coinbase/onchainkit';
 // ---cut-after---
 ```
 
-<img alt="Customizing the name and logo"
-src="https://onchainkit.xyz/assets/commerce-4.png"
-height="364"/>
+<div style={{ display: 'flex', justifyContent: 'center' }}>
+  <img 
+    alt="Add name and logo"
+    src="https://onchainkit.xyz/assets/commerce-4.png"
+    style={{ height: '400px' }}
```

Comment:
if you're only going to display this at 400px, could upload a smaller image

Reply:
done